### ISSUE TITLE
AddObject alternate parameter names

### ISSUE THREAD

**Issue by logankp** [restsharp/RestSharp#924]
It would be nice if AddObject() would support adding parameters with different names than the property (for example, using an attribute such as SerializeAs).

I would be willing to create the PR for this if acceptable.

Thanks!

**Comment by vladonemo** [comment]
I'd also find it useful. And I'm also wiling to provide the PR :)

**Comment by logankp** [comment]
I'm wondering if this project is still actively maintained... No commits for 10 months and many issues gone unanswered.

**Comment by ericmaro** [comment]
here is what i did
`request.AddObject(
               new User
               {
                   email = email,
                   password = password,

               });`

i made a public class
` public class User
        {
            public int Id { get; set; }
            public string email { get; set; }
            public string password { get; set; }
        }`

**Comment by alexeyzimarev** [comment]
I don’t understand the issue to be honest.

@logankp this is a community-driven open source project. It would be great if people will contribute their time and skills to fix issues instead asking why no one is fixing issues for them. /end of rant

**Comment by logankp** [comment]
@alexeyzimarev I meant no disrespect by my original comment (and even said I would be willing to provide a PR). The contributing guidelines specifically say to discuss features before submitting them (which I did in Google groups). After getting no response, I opened an issue. After no further response, I posted my question about project maintenance. And finally two years later I got a rant for a response with the issue being closed.

I'm not sure where you got that I was expecting the issue to be fixed for me.
